build.gradle

```
@@ -57,7 +57,7 @@ subprojects {
     sourceCompatibility = 1.8
 
     tasks.withType(JavaCompile) {
-        options.compilerArgs += ['-Werror', '-Xlint:deprecation']
+        options.compilerArgs += ['-Xlint:deprecation']
```

```suggestion
        options.compilerArgs += ['-Werror', '-Xlint:-deprecation']
```

I think this should get rid of the deprecation warnings while still allowing us to use `-Werror`. Can you check if this works on CI?

---

Done. I get the following message on my build:
"Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details."